digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/ServiceRoadAPI.scala

```diff
@@ -23,14 +52,40 @@ class ServiceRoadAPI(val maintenanceService: MaintenanceService, val roadLinkSer
     basicAuth
   }
 
-  get("/huoltotiet"){
+  val getServiceRoadByBoundingBox =
+    (apiOperation[List[serviceRoadApiResponseOnGetExample]]("getServiceRoadByBoundingBox")
+      tags "Service Road API (Huoltotie API)"
+      summary "Returns all Huoltotie assets inside bounding box. Can be used to get all assets on the UI map area."
```

Comment:
Instead of `Huoltotie`on the summary, it should be written `service road` for coherence